src/amrex/space1d/__init__.py

```diff
@@ -32,3 +32,9 @@
 #
 def d_decl(x, y, z):
     return (x,)
+
+
+def Print(*args, **kwargs):
+    """Wrap amrex::Print() - only the IO processor writes"""
+    if ParallelDescriptor.IOProcessor():
```

Comment:
These bindings to `ParallelProcessor` segfault if they are called before AMReX initialize.

I wonder if we should either in AMReX or at least in pyAMReX add a test for `amrex::initialized` and then do something cleaner (throw or all print).

Reply:
pushed an update